learning_journal/form.py

```diff
@@ -0,0 +1,11 @@
+from wtforms import Form, BooleanField, StringField, validators, TextAreaField
+
+
+class JournalForm(Form):
+
+    """Create a form for our learning journal entries."""
+    title = StringField('title', [validators.Length(min=1, max=128)])
+    text = TextAreaField('text')
+
+
+form = JournalForm()
```

Comment:
I'm not sure why you are doing this here.  You probably actually want to instantiate the form only when you are using it in a view.